script.js

```diff
@@ -0,0 +1,134 @@
+/*Craig Bergmeyer
+Script.js
+INFO 2134
+Thoendel
+March 13, 2020*/
+
+
+window.addEventListener('load', function() {
+    /* Write your solution between this comment */
+    
+    //for loop to get ages 1 through 100
+    for (let age = 1 ; age < 101 ; age++) {
+    	
+    	// creates new option element
+    	let newOption = document.createElement('option');
+		
+  		// enters the age into the new element 
+  		let addAge = document.createTextNode(age); 
+  		// add the text node to the option element
+ 		newOption.appendChild(addAge); 
+  		//sets the position to insert the text
+  		let insertOption = document.getElementById("age");
+		//inserts the text into the select box
+		insertOption.add(newOption);
+ 
+		}
+		
+		
+	
+
+	//Defines the proper submit box and calls the 'onClick' function
+	let submitBox = document.getElementsByTagName('input')[2];
+	submitBox.addEventListener('click', onClick);   
+    	
+	function onClick(e) {
+   	//Removes the error message text.
+   	var eHeader = document.getElementById('errorHeader');
+	if (eHeader != null)
+	   	{
+	    let toDels = document.querySelectorAll('li');
+		for (let toDel of toDels) 
+		{
+	    var eList = document.getElementById('errorList');	  		
+	    eList.parentNode.removeChild(eList);
+	    }
+	    let eHeader = document.getElementById('errorHeader');
+	    eHeader.parentNode.removeChild(eHeader);
+	}
```

Comment:
indentation is a little funky here. Also - error-holder is the ID of the div we want to empty. You could just set its innerHTML to "" and skip all this complicated (though well executed 😄) looping.